word/views.py

```diff
@@ -41,28 +41,56 @@ def get_context_data(self, **kwargs):
         return context
 
     def post(self, request, *args, **kwargs):
+        uploaded_file_url = None
+        if request.method == "POST" and "myfile" in request.FILES:
+            myfile = request.FILES["myfile"]
+            fs = FileSystemStorage()
+            filename = fs.save(myfile.name, myfile)
+            if filename.endswith(".mp3") or filename.endswith(".wma") or\

```

Comment:
This might look a bit prettier with a regex

Reply:
ok. Also, even though It does not store the path into the database, it is storing the files that are not audio in media. perhaps I should change that.